cpp/merge_print_and_online.cc

```diff
@@ -837,8 +837,12 @@ void MergeRecordPair(MARC::Record * const merge_record, MARC::Record * const imp
         if (GetFuzzyIdenticalField(*merge_record, import_field, &merge_field_pos, compare_indicators, compare_subfields))
             continue;
 
-        // Non-existing fields in the merge_record can be inserted unconditionally
-        if (merge_record->findTag(import_field.getTag()) == merge_record->end()) {
+        // Non-existing or 936rv fields in the merge_record can be inserted unconditionally.
```

Comment:
Isn't the "Non-existing fields in the merge_record can be inserted unconditionally" rule unconditionally valid and this logic belongs to the MergeFieldPair936-function below?